source/index.html.md

```diff
@@ -15,4 +15,7 @@ Collecting a list of government APIs will help us understand:
 * where API development is taking place
 * whether APIs are suitable for reuse
 
-If you have any questions, please contact: <api-standards-request@digital.cabinet-office.gov.uk>
+and will help users across government find and access inter-departmental data exchanges.
+
+If you have any questions, please contact: <api-catalogue@digital.cabinet-office.gov.uk>
```

Comment:
I wonder if there should be a full-stop at the end of this line, because the following sentence starts on the same line:

<img width="768" alt="Screenshot 2020-09-25 at 16 04 32" src="https://user-images.githubusercontent.com/4125410/94283380-d5ec9000-ff48-11ea-854b-58b429c8f0c2.png">

Or we could switch these two senteces to be a bullet list.